libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -605,69 +598,62 @@ AP_Airspeed::CalibrationState AP_Airspeed::get_calibration_state() const
 }
 
 // read one airspeed sensor
-void AP_Airspeed::read(uint8_t i)
+void AP_Airspeed_Backend::update()
```

Comment:
i'm not a fan of moving the update to the backend class, it means the backend class is dealing with calibration, offsets, limits etc, which I don't think should be part of the backend